apps/dg/libraries/slickgrid/plugins/slick.rowselectionmodel.js

```diff
@@ -34,10 +34,12 @@
 
     function wrapHandler(handler) {
       return function () {
+        var result;
```

Comment:
The wrapHandler() in DG.CaseTableView.subscribe() was the one that I needed to fix for things to work. The other(s) would just be for consistency in case they ever are used by clients that require the return value. Since we don't seem to be using Slick.RowSelectionModel, it seems to me it should be removed for all of the reasons we usually remove unused code. There are lots of pieces of SlickGrid that we don't include in CODAP because we're not using them.